wheel build for headless fails on 4.7.0 for Py38

### Expected behaviour

When building manually as per [documentation](https://github.com/opencv/opencv-python#manual-builds), I expect to get a .whl file which I can then `pip install`.

(I know I can also install via PyPI on my platform, but I have to build my own wheels.)

### Actual behaviour

However, step 5 (`pip wheel`) fails with a strange error message:

<details><summary>full build log</summary>
<p>

```
Looking in indexes: https://pypi.org/simple, https://pypi.ngc.nvidia.com
Processing ./opencv-python
  Running command pip subprocess to install build dependencies
  Looking in indexes: https://pypi.org/simple, https://pypi.ngc.nvidia.com, https://pypi.ngc.nvidia.com
...
  Installing collected packages: cmake, wheel, tomli, setuptools, pip, packaging, numpy, distro, scikit-build
  Successfully installed cmake-3.26.3 distro-1.8.0 numpy-1.17.3 packaging-23.1 pip-23.1.1 scikit-build-0.17.2 setuptools-59.2.0 tomli-2.0.1 wheel-0.37.0
  Installing build dependencies ... done
  Running command Getting requirements to build wheel
  running egg_info
  creating opencv_python_headless.egg-info
  writing opencv_python_headless.egg-info/PKG-INFO
  writing dependency_links to opencv_python_headless.egg-info/dependency_links.txt
  writing requirements to opencv_python_headless.egg-info/requires.txt
  writing top-level names to opencv_python_headless.egg-info/top_level.txt
  writing manifest file 'opencv_python_headless.egg-info/SOURCES.txt'
  reading manifest file 'opencv_python_headless.egg-info/SOURCES.txt'
  reading manifest template 'MANIFEST.in'
  adding license file 'LICENSE-3RD-PARTY.txt'
  adding license file 'LICENSE.txt'
  writing manifest file 'opencv_python_headless.egg-info/SOURCES.txt'
  /tmp/pip-build-env-2l3nyybo/overlay/lib/python3.8/site-packages/setuptools/dist.py:501: UserWarning: The version specified ('4.7.0.6b73d90') is an invalid version, this may not work as expected with newer versions of setuptools, pip, and PyPI. Please see PEP 440 for more details.
    warnings.warn(
  /tmp/pip-build-env-2l3nyybo/overlay/lib/python3.8/site-packages/pkg_resources/__init__.py:116: PkgResourcesDeprecationWarning: 4.7.0.6b73d90 is an invalid version and will not be supported in a future release
    warnings.warn(
  Getting requirements to build wheel ... done
  Running command Preparing metadata (pyproject.toml)
  running dist_info
  creating /tmp/pip-modern-metadata-qqodxyw_/opencv_python_headless.egg-info
  writing /tmp/pip-modern-metadata-qqodxyw_/opencv_python_headless.egg-info/PKG-INFO
  writing dependency_links to /tmp/pip-modern-metadata-qqodxyw_/opencv_python_headless.egg-info/dependency_links.txt
  writing requirements to /tmp/pip-modern-metadata-qqodxyw_/opencv_python_headless.egg-info/requires.txt
  writing top-level names to /tmp/pip-modern-metadata-qqodxyw_/opencv_python_headless.egg-info/top_level.txt
  writing manifest file '/tmp/pip-modern-metadata-qqodxyw_/opencv_python_headless.egg-info/SOURCES.txt'
  reading manifest file '/tmp/pip-modern-metadata-qqodxyw_/opencv_python_headless.egg-info/SOURCES.txt'
  reading manifest template 'MANIFEST.in'
  adding license file 'LICENSE-3RD-PARTY.txt'
  adding license file 'LICENSE.txt'
  writing manifest file '/tmp/pip-modern-metadata-qqodxyw_/opencv_python_headless.egg-info/SOURCES.txt'
  creating '/tmp/pip-modern-metadata-qqodxyw_/opencv_python_headless.dist-info'
  adding license file "LICENSE.txt" (matched pattern "LICEN[CS]E*")
  adding license file "LICENSE-3RD-PARTY.txt" (matched pattern "LICEN[CS]E*")
  /tmp/pip-build-env-2l3nyybo/overlay/lib/python3.8/site-packages/setuptools/dist.py:501: UserWarning: The version specified ('4.7.0.6b73d90') is an invalid version, this may not work as expected with newer versions of setuptools, pip, and PyPI. Please see PEP 440 for more details.
    warnings.warn(
  /tmp/pip-build-env-2l3nyybo/overlay/lib/python3.8/site-packages/pkg_resources/__init__.py:116: PkgResourcesDeprecationWarning: 4.7.0.6b73d90 is an invalid version and will not be supported in a future release
    warnings.warn(
  Preparing metadata (pyproject.toml) ... done
Collecting numpy>=1.17.0 (from opencv-python-headless===4.7.0.6b73d90)
  Downloading numpy-1.24.3-cp38-cp38-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (17.3 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 17.3/17.3 MB 91.2 MB/s eta 0:00:00
Saved ./numpy-1.24.3-cp38-cp38-manylinux_2_17_x86_64.manylinux2014_x86_64.whl
Building wheels for collected packages: opencv-python-headless
  Running command Building wheel for opencv-python-headless (pyproject.toml)


  --------------------------------------------------------------------------------
  -- Trying 'Ninja' generator
  --------------------------------
  ---------------------------
  ----------------------
  -----------------
  ------------
  -------
  --
  CMake Error: CMake was unable to find a build program corresponding to "Ninja".  CMAKE_MAKE_PROGRAM is not set.  You probably need to select a different build tool.Not searching for unused variables given on the command line.

  -- Configuring incomplete, errors occurred!
  --
  -------
  ------------
  -----------------
  ----------------------
  ---------------------------
  --------------------------------
  -- Trying 'Ninja' generator - failure
  --------------------------------------------------------------------------------



  --------------------------------------------------------------------------------
  -- Trying 'Unix Makefiles' generator
  --------------------------------
  ---------------------------
  ----------------------
  -----------------
  ------------
  -------
  --
  Not searching for unused variables given on the command line.
  -- The C compiler identification is GNU 9.4.0
  -- Detecting C compiler ABI info
  -- Detecting C compiler ABI info - done
  -- Check for working C compiler: /usr/bin/cc - skipped
  -- Detecting C compile features
  -- Detecting C compile features - done
  -- The CXX compiler identification is GNU 9.4.0
  -- Detecting CXX compiler ABI info
  -- Detecting CXX compiler ABI info - done
  -- Check for working CXX compiler: /usr/bin/c++ - skipped
  -- Detecting CXX compile features
  -- Detecting CXX compile features - done
  -- Configuring done (0.4s)
  -- Generating done (0.0s)
  -- Build files have been written to: /test/opencv-python/_cmake_test_compile/build
  --
  -------
  ------------
  -----------------
  ----------------------
  ---------------------------
  --------------------------------
  -- Trying 'Unix Makefiles' generator - success
  --------------------------------------------------------------------------------

  Configuring Project
    Working directory:
      /test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build
    Command:
      /tmp/pip-build-env-2l3nyybo/overlay/lib/python3.8/site-packages/cmake/data/bin/cmake /test/opencv-python/opencv -G 'Unix Makefiles' --no-warn-unused-cli -DCMAKE_INSTALL_PREFIX:PATH=/test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-install -DPYTHON_VERSION_STRING:STRING=3.8.10 -DSKBUILD:INTERNAL=TRUE -DCMAKE_MODULE_PATH:PATH=/tmp/pip-build-env-2l3nyybo/overlay/lib/python3.8/site-packages/skbuild/resources/cmake -DPYTHON_EXECUTABLE:PATH=/test/venv/bin/python3 -DPYTHON_INCLUDE_DIR:PATH=/usr/include/python3.8 -DPYTHON_LIBRARY:PATH=/usr/lib/x86_64-linux-gnu/libpython3.8.so -DPython_EXECUTABLE:PATH=/test/venv/bin/python3 -DPython_ROOT_DIR:PATH=/test/venv -DPython_FIND_REGISTRY:STRING=NEVER -DPython_INCLUDE_DIR:PATH=/usr/include/python3.8 -DPython_NumPy_INCLUDE_DIRS:PATH=/tmp/pip-build-env-2l3nyybo/overlay/lib/python3.8/site-packages/numpy/core/include -DPython3_EXECUTABLE:PATH=/test/venv/bin/python3 -DPython3_ROOT_DIR:PATH=/test/venv -DPython3_FIND_REGISTRY:STRING=NEVER -DPython3_INCLUDE_DIR:PATH=/usr/include/python3.8 -DPython3_NumPy_INCLUDE_DIRS:PATH=/tmp/pip-build-env-2l3nyybo/overlay/lib/python3.8/site-packages/numpy/core/include -DPYTHON3_EXECUTABLE=/test/venv/bin/python3 -DPYTHON3_INCLUDE_DIR=/usr/include/python3.8 -DPYTHON3_LIBRARY=/usr/lib/x86_64-linux-gnu/libpython3.8.so -DBUILD_opencv_python3=ON -DBUILD_opencv_python2=OFF -DBUILD_opencv_java=OFF -DOPENCV_PYTHON3_INSTALL_PATH=python -DINSTALL_CREATE_DISTRIB=ON -DBUILD_opencv_apps=OFF -DBUILD_opencv_freetype=OFF -DBUILD_SHARED_LIBS=OFF -DBUILD_TESTS=OFF -DBUILD_PERF_TESTS=OFF -DBUILD_DOCS=OFF -DPYTHON3_LIMITED_API=ON -DBUILD_OPENEXR=ON -DWITH_WIN32UI=OFF -DWITH_QT=OFF -DWITH_GTK=OFF -DCMAKE_BUILD_TYPE:STRING=Release

  Not searching for unused variables given on the command line.
  -- The CXX compiler identification is GNU 9.4.0
  -- The C compiler identification is GNU 9.4.0
  -- Detecting CXX compiler ABI info
  -- Detecting CXX compiler ABI info - done
  -- Check for working CXX compiler: /usr/bin/c++ - skipped
  -- Detecting CXX compile features
  -- Detecting CXX compile features - done
  -- Detecting C compiler ABI info
  -- Detecting C compiler ABI info - done
  -- Check for working C compiler: /usr/bin/cc - skipped
  -- Detecting C compile features
  -- Detecting C compile features - done
  -- Detected processor: x86_64
  -- Found PythonInterp: /test/venv/bin/python3 (found suitable version "3.8.10", minimum required is "2.7")
  CMake Warning at cmake/OpenCVDetectPython.cmake:81 (message):
    CMake's 'find_host_package(PythonInterp 2.7)' found wrong Python version:

    PYTHON_EXECUTABLE=/test/venv/bin/python3

    PYTHON_VERSION_STRING=3.8.10

    Consider providing the 'PYTHON2_EXECUTABLE' variable via CMake command line
    or environment variables

  Call Stack (most recent call first):
    cmake/OpenCVDetectPython.cmake:271 (find_python)
    CMakeLists.txt:643 (include)

  
  -- Found Python2: /usr/bin/python2.7 (found version "2.7.18") found components: Interpreter
  -- Found PythonInterp: /usr/bin/python2.7 (found version "2.7.18")
  -- Could NOT find PythonLibs: Found unsuitable version "3.8.10", but required is exact version "2.7.18" (found /usr/lib/x86_64-linux-gnu/libpython3.8.so)
  Error processing line 1 of /test/venv/lib/python3.8/site-packages/distutils-precedence.pth:

    Traceback (most recent call last):
      File "/usr/lib/python2.7/site.py", line 158, in addpackage
        exec line
      File "<string>", line 1, in <module>
      File "/test/venv/lib/python3.8/site-packages/_distutils_hack/__init__.py", line 194
        f'spec_for_{name}',
                         ^
    SyntaxError: invalid syntax

  Remainder of file ignored
  Error processing line 1 of /test/venv/lib/python3.8/site-packages/distutils-precedence.pth:

    Traceback (most recent call last):
      File "/usr/lib/python2.7/site.py", line 158, in addpackage
        exec line
      File "<string>", line 1, in <module>
      File "/test/venv/lib/python3.8/site-packages/_distutils_hack/__init__.py", line 194
        f'spec_for_{name}',
                         ^
    SyntaxError: invalid syntax

  Remainder of file ignored
  Traceback (most recent call last):
    File "<string>", line 1, in <module>
    File "/tmp/pip-build-env-2l3nyybo/overlay/lib/python3.8/site-packages/numpy/__init__.py", line 142, in <module>
      from . import core
    File "/tmp/pip-build-env-2l3nyybo/overlay/lib/python3.8/site-packages/numpy/core/__init__.py", line 47, in <module>
      raise ImportError(msg)
  ImportError:

  IMPORTANT: PLEASE READ THIS FOR ADVICE ON HOW TO SOLVE THIS ISSUE!

  Importing the numpy c-extensions failed.
  - Try uninstalling and reinstalling numpy.
  - If you have already done that, then:
    1. Check that you expected to use Python2.7 from "/usr/bin/python2.7",
       and that you have no directories in your PATH or PYTHONPATH that can
       interfere with the Python and numpy version "1.17.3" you're trying to use.
    2. If (1) looks fine, you can open a new issue at
       https://github.com/numpy/numpy/issues.  Please include details on:
       - how you installed Python
       - how you installed numpy
       - your operating system
       - whether or not you have multiple versions of Python installed
       - if you built from source, your compiler versions and ideally a build log

  - If you're working with a numpy git repository, try `git clean -xdf`
    (removes all files not under version control) and rebuild numpy.

  Note: this error has many possible causes, so please don't comment on
  an existing issue about this - open a new one instead.

  Original error was: No module named _multiarray_umath

  -- Found PythonInterp: /test/venv/bin/python3 (found suitable version "3.8.10", minimum required is "3.2")
  -- Found PythonLibs: /usr/lib/x86_64-linux-gnu/libpython3.8.so (found suitable exact version "3.8.10")
  -- Looking for ccache - not found
  -- Performing Test HAVE_CXX_FSIGNED_CHAR
  -- Performing Test HAVE_CXX_FSIGNED_CHAR - Success
...
  --
  -- General configuration for OpenCV 4.7.0 =====================================
  --   Version control:               4.7.0
  --
  --   Platform:
  --     Timestamp:                   2023-04-25T16:33:28Z
  --     Host:                        Linux 5.4.0-147-generic x86_64
  --     CMake:                       3.26.3
  --     CMake generator:             Unix Makefiles
  --     CMake build tool:            /usr/bin/make
  --     Configuration:               Release
  --
  --   CPU/HW features:
  --     Baseline:                    SSE SSE2 SSE3
  --       requested:                 SSE3
  --     Dispatched code generation:  SSE4_1 SSE4_2 FP16 AVX AVX2 AVX512_SKX
  --       requested:                 SSE4_1 SSE4_2 AVX FP16 AVX2 AVX512_SKX
  --       SSE4_1 (16 files):         + SSSE3 SSE4_1
  --       SSE4_2 (1 files):          + SSSE3 SSE4_1 POPCNT SSE4_2
  --       FP16 (0 files):            + SSSE3 SSE4_1 POPCNT SSE4_2 FP16 AVX
  --       AVX (4 files):             + SSSE3 SSE4_1 POPCNT SSE4_2 AVX
  --       AVX2 (32 files):           + SSSE3 SSE4_1 POPCNT SSE4_2 FP16 FMA3 AVX AVX2
  --       AVX512_SKX (5 files):      + SSSE3 SSE4_1 POPCNT SSE4_2 FP16 FMA3 AVX AVX2 AVX_512F AVX512_COMMON AVX512_SKX
  --
  --   C/C++:
  --     Built as dynamic libs?:      NO
  --     C++ standard:                11
  --     C++ Compiler:                /usr/bin/c++  (ver 9.4.0)
  --     C++ flags (Release):         -fsigned-char -W -Wall -Wreturn-type -Wnon-virtual-dtor -Waddress -Wsequence-point -Wformat -Wformat-security -Wmissing-declarations -Wundef -Winit-self -Wpointer-arith -Wshadow -Wsign-promo -Wuninitialized -Wsuggest-override -Wno-delete-non-virtual-dtor -Wno-comment -Wimplicit-fallthrough=3 -Wno-strict-overflow -fdiagnostics-show-option -Wno-long-long -pthread -fomit-frame-pointer -ffunction-sections -fdata-sections  -msse -msse2 -msse3 -fvisibility=hidden -fvisibility-inlines-hidden -O3 -DNDEBUG  -DNDEBUG
  --     C++ flags (Debug):           -fsigned-char -W -Wall -Wreturn-type -Wnon-virtual-dtor -Waddress -Wsequence-point -Wformat -Wformat-security -Wmissing-declarations -Wundef -Winit-self -Wpointer-arith -Wshadow -Wsign-promo -Wuninitialized -Wsuggest-override -Wno-delete-non-virtual-dtor -Wno-comment -Wimplicit-fallthrough=3 -Wno-strict-overflow -fdiagnostics-show-option -Wno-long-long -pthread -fomit-frame-pointer -ffunction-sections -fdata-sections  -msse -msse2 -msse3 -fvisibility=hidden -fvisibility-inlines-hidden -g  -O0 -DDEBUG -D_DEBUG
  --     C Compiler:                  /usr/bin/cc
  --     C flags (Release):           -fsigned-char -W -Wall -Wreturn-type -Waddress -Wsequence-point -Wformat -Wformat-security -Wmissing-declarations -Wmissing-prototypes -Wstrict-prototypes -Wundef -Winit-self -Wpointer-arith -Wshadow -Wuninitialized -Wno-comment -Wimplicit-fallthrough=3 -Wno-strict-overflow -fdiagnostics-show-option -Wno-long-long -pthread -fomit-frame-pointer -ffunction-sections -fdata-sections  -msse -msse2 -msse3 -fvisibility=hidden -O3 -DNDEBUG  -DNDEBUG
  --     C flags (Debug):             -fsigned-char -W -Wall -Wreturn-type -Waddress -Wsequence-point -Wformat -Wformat-security -Wmissing-declarations -Wmissing-prototypes -Wstrict-prototypes -Wundef -Winit-self -Wpointer-arith -Wshadow -Wuninitialized -Wno-comment -Wimplicit-fallthrough=3 -Wno-strict-overflow -fdiagnostics-show-option -Wno-long-long -pthread -fomit-frame-pointer -ffunction-sections -fdata-sections  -msse -msse2 -msse3 -fvisibility=hidden -g  -O0 -DDEBUG -D_DEBUG
  --     Linker flags (Release):      -Wl,--exclude-libs,libippicv.a -Wl,--exclude-libs,libippiw.a   -Wl,--gc-sections -Wl,--as-needed -Wl,--no-undefined
  --     Linker flags (Debug):        -Wl,--exclude-libs,libippicv.a -Wl,--exclude-libs,libippiw.a   -Wl,--gc-sections -Wl,--as-needed -Wl,--no-undefined
  --     ccache:                      NO
  --     Precompiled headers:         NO
  --     Extra dependencies:          /usr/lib/x86_64-linux-gnu/libjpeg.so /usr/lib/x86_64-linux-gnu/libwebp.so /usr/lib/x86_64-linux-gnu/libpng.so /usr/lib/x86_64-linux-gnu/libtiff.so /usr/lib/x86_64-linux-gnu/libz.so dl m pthread rt
  --     3rdparty dependencies:       libprotobuf ade ittnotify libopenjp2 IlmImf quirc ippiw ippicv
  --
  --   OpenCV modules:
  --     To be built:                 calib3d core dnn features2d flann gapi highgui imgcodecs imgproc ml objdetect photo python3 stitching video videoio
  --     Disabled:                    world
  --     Disabled by dependency:      -
  --     Unavailable:                 java python2 ts
  --     Applications:                -
  --     Documentation:               NO
  --     Non-free algorithms:         NO
  --
  --   GUI:                           NONE
  --     VTK support:                 NO
  --
  --   Media I/O:
  --     ZLib:                        /usr/lib/x86_64-linux-gnu/libz.so (ver 1.2.11)
  --     JPEG:                        /usr/lib/x86_64-linux-gnu/libjpeg.so (ver 80)
  --     WEBP:                        /usr/lib/x86_64-linux-gnu/libwebp.so (ver encoder: 0x020e)
  --     PNG:                         /usr/lib/x86_64-linux-gnu/libpng.so (ver 1.6.37)
  --     TIFF:                        /usr/lib/x86_64-linux-gnu/libtiff.so (ver 42 / 4.1.0)
  --     JPEG 2000:                   build (ver 2.4.0)
  --     OpenEXR:                     build (ver 2.3.0)
  --     HDR:                         YES
  --     SUNRASTER:                   YES
  --     PXM:                         YES
  --     PFM:                         YES
  --
  --   Video I/O:
  --     DC1394:                      NO
  --     FFMPEG:                      NO
  --       avcodec:                   NO
  --       avformat:                  NO
  --       avutil:                    NO
  --       swscale:                   NO
  --       avresample:                NO
  --     GStreamer:                   NO
  --     v4l/v4l2:                    YES (linux/videodev2.h)
  --
  --   Parallel framework:            pthreads
  --
  --   Trace:                         YES (with Intel ITT)
  --
  --   Other third-party libraries:
  --     Intel IPP:                   2020.0.0 Gold [2020.0.0]
  --            at:                   /test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build/3rdparty/ippicv/ippicv_lnx/icv
  --     Intel IPP IW:                sources (2020.0.0)
  --               at:                /test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build/3rdparty/ippicv/ippicv_lnx/iw
  --     VA:                          NO
  --     Lapack:                      NO
  --     Eigen:                       NO
  --     Custom HAL:                  NO
  --     Protobuf:                    build (3.19.1)
  --
  --   OpenCL:                        YES (no extra features)
  --     Include path:                /test/opencv-python/opencv/3rdparty/include/opencl/1.2
  --     Link libraries:              Dynamic load
  --
  --   Python 3:
  --     Interpreter:                 /test/venv/bin/python3 (ver 3.8.10)
  --     Libraries:                   /usr/lib/x86_64-linux-gnu/libpython3.8.so (ver 3.8.10)
  --     numpy:                       /tmp/pip-build-env-2l3nyybo/overlay/lib/python3.8/site-packages/numpy/core/include (ver 1.17.3)
  --     install path:                python/cv2/python-3
  --
  --   Python (for build):            /usr/bin/python2.7
  --
  --   Java:
  --     ant:                         NO
  --     JNI:                         NO
  --     Java wrappers:               NO
  --     Java tests:                  NO
  --
  --   Install to:                    /test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-install
  -- -----------------------------------------------------------------
  --
  -- Configuring done (13.3s)
  -- Generating done (0.1s)
  -- Build files have been written to: /test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build
...
   [ 98%] Linking CXX static library ../../lib/libopencv_gapi.a
  make[3]: Leaving directory '/test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build'
  [ 98%] Built target opencv_gapi
  make[3]: Entering directory '/test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build'
  make[3]: Leaving directory '/test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build'
  make[3]: Entering directory '/test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build'
  [ 99%] Generate files for Python bindings and documentation
  Error processing line 1 of /test/venv/lib/python3.8/site-packages/distutils-precedence.pth:

    Traceback (most recent call last):
      File "/usr/lib/python2.7/site.py", line 158, in addpackage
        exec line
      File "<string>", line 1, in <module>
      File "/test/venv/lib/python3.8/site-packages/_distutils_hack/__init__.py", line 194
        f'spec_for_{name}',
                         ^
    SyntaxError: invalid syntax

  Remainder of file ignored
  Note: Class cv::Feature2D has more than 1 base class (not supported by Python C extensions)
        Bases:  cv::Algorithm, cv::class, cv::Feature2D, cv::Algorithm
        Only the first base class will be used
  Note: Class cv::detail::GraphCutSeamFinder has more than 1 base class (not supported by Python C extensions)
        Bases:  cv::detail::GraphCutSeamFinderBase, cv::detail::SeamFinder
        Only the first base class will be used
  make[3]: Leaving directory '/test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build'
  [ 99%] Built target gen_opencv_python_source
  make[3]: Entering directory '/test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build'
  make[3]: Leaving directory '/test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build'
  make[3]: Entering directory '/test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build'
  [ 99%] Building CXX object modules/python3/CMakeFiles/opencv_python3.dir/__/src2/cv2.cpp.o
  [100%] Building CXX object modules/python3/CMakeFiles/opencv_python3.dir/__/src2/cv2_util.cpp.o
  [100%] Building CXX object modules/python3/CMakeFiles/opencv_python3.dir/__/src2/cv2_numpy.cpp.o
  [100%] Building CXX object modules/python3/CMakeFiles/opencv_python3.dir/__/src2/cv2_convert.cpp.o
  [100%] Building CXX object modules/python3/CMakeFiles/opencv_python3.dir/__/src2/cv2_highgui.cpp.o
  [100%] Linking CXX shared module ../../lib/python3/cv2.abi3.so
  make[3]: Leaving directory '/test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build'
  [100%] Built target opencv_python3
  make[2]: Leaving directory '/test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build'
  Install the project...
  -- Install configuration: "Release"
  -- Installing: /test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-install/share/licenses/opencv4/ippicv-readme.htm
  -- Installing: /test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-install/share/licenses/opencv4/ippicv-EULA.txt
  -- Installing: /test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-install/share/licenses/opencv4/ippicv-third-party-programs.txt
  -- Installing: /test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-install/lib/opencv4/3rdparty/libippicv.a
  -- Installing: /test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-install/share/licenses/opencv4/ippiw-support.txt
  -- Installing: /test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-install/share/licenses/opencv4/ippiw-third-party-programs.txt
  -- Installing: /test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-install/share/licenses/opencv4/ippiw-EULA.txt
  -- Installing: /test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-install/share/licenses/opencv4/opencl-headers-LICENSE.txt
  -- Installing: /test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-install/lib/opencv4/3rdparty/libade.a
  -- Installing: /test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-install/share/licenses/opencv4/ade-LICENSE
  -- Installing: /test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-install/include/opencv4/opencv2/cvconfig.h
...
  -- Installing: /test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-install/share/opencv4/lbpcascades/lbpcascade_silverware.xml
  make[1]: Leaving directory '/test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build'
  Copying files from CMake output
  creating directory _skbuild/linux-x86_64-3.8/cmake-install/cv2
  copying _skbuild/linux-x86_64-3.8/cmake-install/python/cv2/python-3/cv2.abi3.so -> _skbuild/linux-x86_64-3.8/cmake-install/cv2/cv2.abi3.so
  copying _skbuild/linux-x86_64-3.8/cmake-install/python/cv2/__init__.py -> _skbuild/linux-x86_64-3.8/cmake-install/cv2/__init__.py
  copying _skbuild/linux-x86_64-3.8/cmake-install/python/cv2/load_config_py2.py -> _skbuild/linux-x86_64-3.8/cmake-install/cv2/load_config_py2.py
  copying _skbuild/linux-x86_64-3.8/cmake-install/python/cv2/load_config_py3.py -> _skbuild/linux-x86_64-3.8/cmake-install/cv2/load_config_py3.py
  copying _skbuild/linux-x86_64-3.8/cmake-install/python/cv2/config.py -> _skbuild/linux-x86_64-3.8/cmake-install/cv2/config.py
  copying _skbuild/linux-x86_64-3.8/cmake-install/python/cv2/config-3.py -> _skbuild/linux-x86_64-3.8/cmake-install/cv2/config-3.py
  creating directory _skbuild/linux-x86_64-3.8/cmake-install/cv2/data
  copying _skbuild/linux-x86_64-3.8/cmake-install/share/opencv4/haarcascades/haarcascade_eye.xml -> _skbuild/linux-x86_64-3.8/cmake-install/cv2/data/haarcascade_eye.xml
  copying _skbuild/linux-x86_64-3.8/cmake-install/share/opencv4/haarcascades/haarcascade_eye_tree_eyeglasses.xml -> _skbuild/linux-x86_64-3.8/cmake-install/cv2/data/haarcascade_eye_tree_eyeglasses.xml
  copying _skbuild/linux-x86_64-3.8/cmake-install/share/opencv4/haarcascades/haarcascade_frontalcatface.xml -> _skbuild/linux-x86_64-3.8/cmake-install/cv2/data/haarcascade_frontalcatface.xml
  copying _skbuild/linux-x86_64-3.8/cmake-install/share/opencv4/haarcascades/haarcascade_frontalcatface_extended.xml -> _skbuild/linux-x86_64-3.8/cmake-install/cv2/data/haarcascade_frontalcatface_extended.xml
  copying _skbuild/linux-x86_64-3.8/cmake-install/share/opencv4/haarcascades/haarcascade_frontalface_alt.xml -> _skbuild/linux-x86_64-3.8/cmake-install/cv2/data/haarcascade_frontalface_alt.xml
  copying _skbuild/linux-x86_64-3.8/cmake-install/share/opencv4/haarcascades/haarcascade_frontalface_alt2.xml -> _skbuild/linux-x86_64-3.8/cmake-install/cv2/data/haarcascade_frontalface_alt2.xml
  copying _skbuild/linux-x86_64-3.8/cmake-install/share/opencv4/haarcascades/haarcascade_frontalface_alt_tree.xml -> _skbuild/linux-x86_64-3.8/cmake-install/cv2/data/haarcascade_frontalface_alt_tree.xml
  copying _skbuild/linux-x86_64-3.8/cmake-install/share/opencv4/haarcascades/haarcascade_frontalface_default.xml -> _skbuild/linux-x86_64-3.8/cmake-install/cv2/data/haarcascade_frontalface_default.xml
  copying _skbuild/linux-x86_64-3.8/cmake-install/share/opencv4/haarcascades/haarcascade_fullbody.xml -> _skbuild/linux-x86_64-3.8/cmake-install/cv2/data/haarcascade_fullbody.xml
  copying _skbuild/linux-x86_64-3.8/cmake-install/share/opencv4/haarcascades/haarcascade_lefteye_2splits.xml -> _skbuild/linux-x86_64-3.8/cmake-install/cv2/data/haarcascade_lefteye_2splits.xml
  copying _skbuild/linux-x86_64-3.8/cmake-install/share/opencv4/haarcascades/haarcascade_license_plate_rus_16stages.xml -> _skbuild/linux-x86_64-3.8/cmake-install/cv2/data/haarcascade_license_plate_rus_16stages.xml
  copying _skbuild/linux-x86_64-3.8/cmake-install/share/opencv4/haarcascades/haarcascade_lowerbody.xml -> _skbuild/linux-x86_64-3.8/cmake-install/cv2/data/haarcascade_lowerbody.xml
  copying _skbuild/linux-x86_64-3.8/cmake-install/share/opencv4/haarcascades/haarcascade_profileface.xml -> _skbuild/linux-x86_64-3.8/cmake-install/cv2/data/haarcascade_profileface.xml
  copying _skbuild/linux-x86_64-3.8/cmake-install/share/opencv4/haarcascades/haarcascade_righteye_2splits.xml -> _skbuild/linux-x86_64-3.8/cmake-install/cv2/data/haarcascade_righteye_2splits.xml
  copying _skbuild/linux-x86_64-3.8/cmake-install/share/opencv4/haarcascades/haarcascade_russian_plate_number.xml -> _skbuild/linux-x86_64-3.8/cmake-install/cv2/data/haarcascade_russian_plate_number.xml
  copying _skbuild/linux-x86_64-3.8/cmake-install/share/opencv4/haarcascades/haarcascade_smile.xml -> _skbuild/linux-x86_64-3.8/cmake-install/cv2/data/haarcascade_smile.xml
  copying _skbuild/linux-x86_64-3.8/cmake-install/share/opencv4/haarcascades/haarcascade_upperbody.xml -> _skbuild/linux-x86_64-3.8/cmake-install/cv2/data/haarcascade_upperbody.xml
  creating directory _skbuild/linux-x86_64-3.8/cmake-install/cv2/gapi
  copying _skbuild/linux-x86_64-3.8/cmake-install/python/cv2/gapi/__init__.py -> _skbuild/linux-x86_64-3.8/cmake-install/cv2/gapi/__init__.py
  creating directory _skbuild/linux-x86_64-3.8/cmake-install/cv2/mat_wrapper
  copying _skbuild/linux-x86_64-3.8/cmake-install/python/cv2/mat_wrapper/__init__.py -> _skbuild/linux-x86_64-3.8/cmake-install/cv2/mat_wrapper/__init__.py
  creating directory _skbuild/linux-x86_64-3.8/cmake-install/cv2/misc
  copying _skbuild/linux-x86_64-3.8/cmake-install/python/cv2/misc/__init__.py -> _skbuild/linux-x86_64-3.8/cmake-install/cv2/misc/__init__.py
  copying _skbuild/linux-x86_64-3.8/cmake-install/python/cv2/misc/version.py -> _skbuild/linux-x86_64-3.8/cmake-install/cv2/misc/version.py
  creating directory _skbuild/linux-x86_64-3.8/cmake-install/cv2/utils
  copying _skbuild/linux-x86_64-3.8/cmake-install/python/cv2/utils/__init__.py -> _skbuild/linux-x86_64-3.8/cmake-install/cv2/utils/__init__.py
  Copying files from non-default sourcetree locations
  copying LICENSE.txt -> _skbuild/linux-x86_64-3.8/cmake-install/cv2/LICENSE.txt
  copying LICENSE-3RD-PARTY.txt -> _skbuild/linux-x86_64-3.8/cmake-install/cv2/LICENSE-3RD-PARTY.txt

  copying cv2/data/__init__.py -> _skbuild/linux-x86_64-3.8/cmake-install/cv2/data/__init__.py
  copying /test/opencv-python/cv2/version.py -> _skbuild/linux-x86_64-3.8/cmake-install/cv2/version.py

  running bdist_wheel
  running build
  running build_py
  creating _skbuild/linux-x86_64-3.8/setuptools/lib.linux-x86_64-3.8
  creating _skbuild/linux-x86_64-3.8/setuptools/lib.linux-x86_64-3.8/cv2
  copying _skbuild/linux-x86_64-3.8/cmake-install/cv2/__init__.py -> _skbuild/linux-x86_64-3.8/setuptools/lib.linux-x86_64-3.8/cv2
  copying _skbuild/linux-x86_64-3.8/cmake-install/cv2/config-3.py -> _skbuild/linux-x86_64-3.8/setuptools/lib.linux-x86_64-3.8/cv2
  copying _skbuild/linux-x86_64-3.8/cmake-install/cv2/version.py -> _skbuild/linux-x86_64-3.8/setuptools/lib.linux-x86_64-3.8/cv2
  copying _skbuild/linux-x86_64-3.8/cmake-install/cv2/load_config_py2.py -> _skbuild/linux-x86_64-3.8/setuptools/lib.linux-x86_64-3.8/cv2
  copying _skbuild/linux-x86_64-3.8/cmake-install/cv2/load_config_py3.py -> _skbuild/linux-x86_64-3.8/setuptools/lib.linux-x86_64-3.8/cv2
  copying _skbuild/linux-x86_64-3.8/cmake-install/cv2/config.py -> _skbuild/linux-x86_64-3.8/setuptools/lib.linux-x86_64-3.8/cv2
  creating _skbuild/linux-x86_64-3.8/setuptools/lib.linux-x86_64-3.8/cv2/data
  copying _skbuild/linux-x86_64-3.8/cmake-install/cv2/data/__init__.py -> _skbuild/linux-x86_64-3.8/setuptools/lib.linux-x86_64-3.8/cv2/data
  copying _skbuild/linux-x86_64-3.8/cmake-install/cv2/LICENSE.txt -> _skbuild/linux-x86_64-3.8/setuptools/lib.linux-x86_64-3.8/cv2
  copying _skbuild/linux-x86_64-3.8/cmake-install/cv2/LICENSE-3RD-PARTY.txt -> _skbuild/linux-x86_64-3.8/setuptools/lib.linux-x86_64-3.8/cv2
  copying _skbuild/linux-x86_64-3.8/cmake-install/cv2/cv2.abi3.so -> _skbuild/linux-x86_64-3.8/setuptools/lib.linux-x86_64-3.8/cv2
  creating _skbuild/linux-x86_64-3.8/setuptools/lib.linux-x86_64-3.8/cv2/gapi
  copying _skbuild/linux-x86_64-3.8/cmake-install/cv2/gapi/__init__.py -> _skbuild/linux-x86_64-3.8/setuptools/lib.linux-x86_64-3.8/cv2/gapi
  creating _skbuild/linux-x86_64-3.8/setuptools/lib.linux-x86_64-3.8/cv2/mat_wrapper
  copying _skbuild/linux-x86_64-3.8/cmake-install/cv2/mat_wrapper/__init__.py -> _skbuild/linux-x86_64-3.8/setuptools/lib.linux-x86_64-3.8/cv2/mat_wrapper
  creating _skbuild/linux-x86_64-3.8/setuptools/lib.linux-x86_64-3.8/cv2/misc
  copying _skbuild/linux-x86_64-3.8/cmake-install/cv2/misc/__init__.py -> _skbuild/linux-x86_64-3.8/setuptools/lib.linux-x86_64-3.8/cv2/misc
  copying _skbuild/linux-x86_64-3.8/cmake-install/cv2/misc/version.py -> _skbuild/linux-x86_64-3.8/setuptools/lib.linux-x86_64-3.8/cv2/misc
  creating _skbuild/linux-x86_64-3.8/setuptools/lib.linux-x86_64-3.8/cv2/utils
  copying _skbuild/linux-x86_64-3.8/cmake-install/cv2/utils/__init__.py -> _skbuild/linux-x86_64-3.8/setuptools/lib.linux-x86_64-3.8/cv2/utils
  copying _skbuild/linux-x86_64-3.8/cmake-install/cv2/data/haarcascade_upperbody.xml -> _skbuild/linux-x86_64-3.8/setuptools/lib.linux-x86_64-3.8/cv2/data
  copying _skbuild/linux-x86_64-3.8/cmake-install/cv2/data/haarcascade_frontalface_default.xml -> _skbuild/linux-x86_64-3.8/setuptools/lib.linux-x86_64-3.8/cv2/data
  copying _skbuild/linux-x86_64-3.8/cmake-install/cv2/data/haarcascade_frontalface_alt.xml -> _skbuild/linux-x86_64-3.8/setuptools/lib.linux-x86_64-3.8/cv2/data
  copying _skbuild/linux-x86_64-3.8/cmake-install/cv2/data/haarcascade_eye.xml -> _skbuild/linux-x86_64-3.8/setuptools/lib.linux-x86_64-3.8/cv2/data
  copying _skbuild/linux-x86_64-3.8/cmake-install/cv2/data/haarcascade_frontalface_alt2.xml -> _skbuild/linux-x86_64-3.8/setuptools/lib.linux-x86_64-3.8/cv2/data
  copying _skbuild/linux-x86_64-3.8/cmake-install/cv2/data/haarcascade_lefteye_2splits.xml -> _skbuild/linux-x86_64-3.8/setuptools/lib.linux-x86_64-3.8/cv2/data
  copying _skbuild/linux-x86_64-3.8/cmake-install/cv2/data/haarcascade_profileface.xml -> _skbuild/linux-x86_64-3.8/setuptools/lib.linux-x86_64-3.8/cv2/data
  copying _skbuild/linux-x86_64-3.8/cmake-install/cv2/data/haarcascade_russian_plate_number.xml -> _skbuild/linux-x86_64-3.8/setuptools/lib.linux-x86_64-3.8/cv2/data
  copying _skbuild/linux-x86_64-3.8/cmake-install/cv2/data/haarcascade_lowerbody.xml -> _skbuild/linux-x86_64-3.8/setuptools/lib.linux-x86_64-3.8/cv2/data
  copying _skbuild/linux-x86_64-3.8/cmake-install/cv2/data/haarcascade_license_plate_rus_16stages.xml -> _skbuild/linux-x86_64-3.8/setuptools/lib.linux-x86_64-3.8/cv2/data
  copying _skbuild/linux-x86_64-3.8/cmake-install/cv2/data/haarcascade_righteye_2splits.xml -> _skbuild/linux-x86_64-3.8/setuptools/lib.linux-x86_64-3.8/cv2/data
  copying _skbuild/linux-x86_64-3.8/cmake-install/cv2/data/haarcascade_eye_tree_eyeglasses.xml -> _skbuild/linux-x86_64-3.8/setuptools/lib.linux-x86_64-3.8/cv2/data
  copying _skbuild/linux-x86_64-3.8/cmake-install/cv2/data/haarcascade_frontalcatface.xml -> _skbuild/linux-x86_64-3.8/setuptools/lib.linux-x86_64-3.8/cv2/data
  copying _skbuild/linux-x86_64-3.8/cmake-install/cv2/data/haarcascade_smile.xml -> _skbuild/linux-x86_64-3.8/setuptools/lib.linux-x86_64-3.8/cv2/data
  copying _skbuild/linux-x86_64-3.8/cmake-install/cv2/data/haarcascade_frontalcatface_extended.xml -> _skbuild/linux-x86_64-3.8/setuptools/lib.linux-x86_64-3.8/cv2/data
  copying _skbuild/linux-x86_64-3.8/cmake-install/cv2/data/haarcascade_fullbody.xml -> _skbuild/linux-x86_64-3.8/setuptools/lib.linux-x86_64-3.8/cv2/data
  copying _skbuild/linux-x86_64-3.8/cmake-install/cv2/data/haarcascade_frontalface_alt_tree.xml -> _skbuild/linux-x86_64-3.8/setuptools/lib.linux-x86_64-3.8/cv2/data
  copied 7 files
  running build_ext
  installing to _skbuild/linux-x86_64-3.8/setuptools/bdist.linux-x86_64/wheel
  running install
  running install_lib
  creating _skbuild/linux-x86_64-3.8/setuptools/bdist.linux-x86_64
  creating _skbuild/linux-x86_64-3.8/setuptools/bdist.linux-x86_64/wheel
  creating _skbuild/linux-x86_64-3.8/setuptools/bdist.linux-x86_64/wheel/cv2
  copying _skbuild/linux-x86_64-3.8/setuptools/lib.linux-x86_64-3.8/cv2/__init__.py -> _skbuild/linux-x86_64-3.8/setuptools/bdist.linux-x86_64/wheel/cv2
  creating _skbuild/linux-x86_64-3.8/setuptools/bdist.linux-x86_64/wheel/cv2/data
  copying _skbuild/linux-x86_64-3.8/setuptools/lib.linux-x86_64-3.8/cv2/data/haarcascade_upperbody.xml -> _skbuild/linux-x86_64-3.8/setuptools/bdist.linux-x86_64/wheel/cv2/data
  copying _skbuild/linux-x86_64-3.8/setuptools/lib.linux-x86_64-3.8/cv2/data/haarcascade_frontalface_default.xml -> _skbuild/linux-x86_64-3.8/setuptools/bdist.linux-x86_64/wheel/cv2/data
  copying _skbuild/linux-x86_64-3.8/setuptools/lib.linux-x86_64-3.8/cv2/data/__init__.py -> _skbuild/linux-x86_64-3.8/setuptools/bdist.linux-x86_64/wheel/cv2/data
  copying _skbuild/linux-x86_64-3.8/setuptools/lib.linux-x86_64-3.8/cv2/data/haarcascade_frontalface_alt.xml -> _skbuild/linux-x86_64-3.8/setuptools/bdist.linux-x86_64/wheel/cv2/data
  copying _skbuild/linux-x86_64-3.8/setuptools/lib.linux-x86_64-3.8/cv2/data/haarcascade_eye.xml -> _skbuild/linux-x86_64-3.8/setuptools/bdist.linux-x86_64/wheel/cv2/data
  copying _skbuild/linux-x86_64-3.8/setuptools/lib.linux-x86_64-3.8/cv2/data/haarcascade_frontalface_alt2.xml -> _skbuild/linux-x86_64-3.8/setuptools/bdist.linux-x86_64/wheel/cv2/data
  copying _skbuild/linux-x86_64-3.8/setuptools/lib.linux-x86_64-3.8/cv2/data/haarcascade_lefteye_2splits.xml -> _skbuild/linux-x86_64-3.8/setuptools/bdist.linux-x86_64/wheel/cv2/data
  copying _skbuild/linux-x86_64-3.8/setuptools/lib.linux-x86_64-3.8/cv2/data/haarcascade_profileface.xml -> _skbuild/linux-x86_64-3.8/setuptools/bdist.linux-x86_64/wheel/cv2/data
  copying _skbuild/linux-x86_64-3.8/setuptools/lib.linux-x86_64-3.8/cv2/data/haarcascade_russian_plate_number.xml -> _skbuild/linux-x86_64-3.8/setuptools/bdist.linux-x86_64/wheel/cv2/data
  copying _skbuild/linux-x86_64-3.8/setuptools/lib.linux-x86_64-3.8/cv2/data/haarcascade_lowerbody.xml -> _skbuild/linux-x86_64-3.8/setuptools/bdist.linux-x86_64/wheel/cv2/data
  copying _skbuild/linux-x86_64-3.8/setuptools/lib.linux-x86_64-3.8/cv2/data/haarcascade_license_plate_rus_16stages.xml -> _skbuild/linux-x86_64-3.8/setuptools/bdist.linux-x86_64/wheel/cv2/data
  copying _skbuild/linux-x86_64-3.8/setuptools/lib.linux-x86_64-3.8/cv2/data/haarcascade_righteye_2splits.xml -> _skbuild/linux-x86_64-3.8/setuptools/bdist.linux-x86_64/wheel/cv2/data
  copying _skbuild/linux-x86_64-3.8/setuptools/lib.linux-x86_64-3.8/cv2/data/haarcascade_eye_tree_eyeglasses.xml -> _skbuild/linux-x86_64-3.8/setuptools/bdist.linux-x86_64/wheel/cv2/data
  copying _skbuild/linux-x86_64-3.8/setuptools/lib.linux-x86_64-3.8/cv2/data/haarcascade_frontalcatface.xml -> _skbuild/linux-x86_64-3.8/setuptools/bdist.linux-x86_64/wheel/cv2/data
  copying _skbuild/linux-x86_64-3.8/setuptools/lib.linux-x86_64-3.8/cv2/data/haarcascade_smile.xml -> _skbuild/linux-x86_64-3.8/setuptools/bdist.linux-x86_64/wheel/cv2/data
  copying _skbuild/linux-x86_64-3.8/setuptools/lib.linux-x86_64-3.8/cv2/data/haarcascade_frontalcatface_extended.xml -> _skbuild/linux-x86_64-3.8/setuptools/bdist.linux-x86_64/wheel/cv2/data
  copying _skbuild/linux-x86_64-3.8/setuptools/lib.linux-x86_64-3.8/cv2/data/haarcascade_fullbody.xml -> _skbuild/linux-x86_64-3.8/setuptools/bdist.linux-x86_64/wheel/cv2/data
  copying _skbuild/linux-x86_64-3.8/setuptools/lib.linux-x86_64-3.8/cv2/data/haarcascade_frontalface_alt_tree.xml -> _skbuild/linux-x86_64-3.8/setuptools/bdist.linux-x86_64/wheel/cv2/data
  copying _skbuild/linux-x86_64-3.8/setuptools/lib.linux-x86_64-3.8/cv2/config-3.py -> _skbuild/linux-x86_64-3.8/setuptools/bdist.linux-x86_64/wheel/cv2
  copying _skbuild/linux-x86_64-3.8/setuptools/lib.linux-x86_64-3.8/cv2/version.py -> _skbuild/linux-x86_64-3.8/setuptools/bdist.linux-x86_64/wheel/cv2
  creating _skbuild/linux-x86_64-3.8/setuptools/bdist.linux-x86_64/wheel/cv2/mat_wrapper
  copying _skbuild/linux-x86_64-3.8/setuptools/lib.linux-x86_64-3.8/cv2/mat_wrapper/__init__.py -> _skbuild/linux-x86_64-3.8/setuptools/bdist.linux-x86_64/wheel/cv2/mat_wrapper
  copying _skbuild/linux-x86_64-3.8/setuptools/lib.linux-x86_64-3.8/cv2/LICENSE.txt -> _skbuild/linux-x86_64-3.8/setuptools/bdist.linux-x86_64/wheel/cv2
  copying _skbuild/linux-x86_64-3.8/setuptools/lib.linux-x86_64-3.8/cv2/load_config_py2.py -> _skbuild/linux-x86_64-3.8/setuptools/bdist.linux-x86_64/wheel/cv2
  creating _skbuild/linux-x86_64-3.8/setuptools/bdist.linux-x86_64/wheel/cv2/misc
  copying _skbuild/linux-x86_64-3.8/setuptools/lib.linux-x86_64-3.8/cv2/misc/__init__.py -> _skbuild/linux-x86_64-3.8/setuptools/bdist.linux-x86_64/wheel/cv2/misc
  copying _skbuild/linux-x86_64-3.8/setuptools/lib.linux-x86_64-3.8/cv2/misc/version.py -> _skbuild/linux-x86_64-3.8/setuptools/bdist.linux-x86_64/wheel/cv2/misc
  copying _skbuild/linux-x86_64-3.8/setuptools/lib.linux-x86_64-3.8/cv2/cv2.abi3.so -> _skbuild/linux-x86_64-3.8/setuptools/bdist.linux-x86_64/wheel/cv2
  copying _skbuild/linux-x86_64-3.8/setuptools/lib.linux-x86_64-3.8/cv2/load_config_py3.py -> _skbuild/linux-x86_64-3.8/setuptools/bdist.linux-x86_64/wheel/cv2
  creating _skbuild/linux-x86_64-3.8/setuptools/bdist.linux-x86_64/wheel/cv2/gapi
  copying _skbuild/linux-x86_64-3.8/setuptools/lib.linux-x86_64-3.8/cv2/gapi/__init__.py -> _skbuild/linux-x86_64-3.8/setuptools/bdist.linux-x86_64/wheel/cv2/gapi
  copying _skbuild/linux-x86_64-3.8/setuptools/lib.linux-x86_64-3.8/cv2/LICENSE-3RD-PARTY.txt -> _skbuild/linux-x86_64-3.8/setuptools/bdist.linux-x86_64/wheel/cv2
  creating _skbuild/linux-x86_64-3.8/setuptools/bdist.linux-x86_64/wheel/cv2/utils
  copying _skbuild/linux-x86_64-3.8/setuptools/lib.linux-x86_64-3.8/cv2/utils/__init__.py -> _skbuild/linux-x86_64-3.8/setuptools/bdist.linux-x86_64/wheel/cv2/utils
  copying _skbuild/linux-x86_64-3.8/setuptools/lib.linux-x86_64-3.8/cv2/config.py -> _skbuild/linux-x86_64-3.8/setuptools/bdist.linux-x86_64/wheel/cv2
  copied 32 files
  running install_egg_info
  running egg_info
  writing opencv_python_headless.egg-info/PKG-INFO
  writing dependency_links to opencv_python_headless.egg-info/dependency_links.txt
  writing requirements to opencv_python_headless.egg-info/requires.txt
  writing top-level names to opencv_python_headless.egg-info/top_level.txt
  reading manifest file 'opencv_python_headless.egg-info/SOURCES.txt'
  reading manifest template 'MANIFEST.in'
  adding license file 'LICENSE-3RD-PARTY.txt'
  adding license file 'LICENSE.txt'
  writing manifest file 'opencv_python_headless.egg-info/SOURCES.txt'
  Copying opencv_python_headless.egg-info to _skbuild/linux-x86_64-3.8/setuptools/bdist.linux-x86_64/wheel/opencv_python_headless-4.7.0.6b73d90-py3.8.egg-info
  running install_scripts
  copied 0 files
  adding license file "LICENSE.txt" (matched pattern "LICEN[CS]E*")
  adding license file "LICENSE-3RD-PARTY.txt" (matched pattern "LICEN[CS]E*")
  creating _skbuild/linux-x86_64-3.8/setuptools/bdist.linux-x86_64/wheel/opencv_python_headless-4.7.0.6b73d90.dist-info/WHEEL
  creating '/tmp/pip-wheel-i9lqdy13/tmptt193jdl/opencv_python_headless-4.7.0.6b73d90-cp38-cp38-linux_x86_64.whl' and adding '_skbuild/linux-x86_64-3.8/setuptools/bdist.linux-x86_64/wheel' to it
  adding 'cv2/LICENSE-3RD-PARTY.txt'
  adding 'cv2/LICENSE.txt'
  adding 'cv2/__init__.py'
  adding 'cv2/config-3.py'
  adding 'cv2/config.py'
  adding 'cv2/cv2.abi3.so'
  adding 'cv2/load_config_py2.py'
  adding 'cv2/load_config_py3.py'
  adding 'cv2/version.py'
  adding 'cv2/data/__init__.py'
  adding 'cv2/data/haarcascade_eye.xml'
  adding 'cv2/data/haarcascade_eye_tree_eyeglasses.xml'
  adding 'cv2/data/haarcascade_frontalcatface.xml'
  adding 'cv2/data/haarcascade_frontalcatface_extended.xml'
  adding 'cv2/data/haarcascade_frontalface_alt.xml'
  adding 'cv2/data/haarcascade_frontalface_alt2.xml'
  adding 'cv2/data/haarcascade_frontalface_alt_tree.xml'
  adding 'cv2/data/haarcascade_frontalface_default.xml'
  adding 'cv2/data/haarcascade_fullbody.xml'
  adding 'cv2/data/haarcascade_lefteye_2splits.xml'
  adding 'cv2/data/haarcascade_license_plate_rus_16stages.xml'
  adding 'cv2/data/haarcascade_lowerbody.xml'
  adding 'cv2/data/haarcascade_profileface.xml'
  adding 'cv2/data/haarcascade_righteye_2splits.xml'
  adding 'cv2/data/haarcascade_russian_plate_number.xml'
  adding 'cv2/data/haarcascade_smile.xml'
  adding 'cv2/data/haarcascade_upperbody.xml'
  adding 'cv2/gapi/__init__.py'
  adding 'cv2/mat_wrapper/__init__.py'
  adding 'cv2/misc/__init__.py'
  adding 'cv2/misc/version.py'
  adding 'cv2/utils/__init__.py'
  adding 'opencv_python_headless-4.7.0.6b73d90.dist-info/LICENSE-3RD-PARTY.txt'
  adding 'opencv_python_headless-4.7.0.6b73d90.dist-info/LICENSE.txt'
  adding 'opencv_python_headless-4.7.0.6b73d90.dist-info/METADATA'
  adding 'opencv_python_headless-4.7.0.6b73d90.dist-info/WHEEL'
  adding 'opencv_python_headless-4.7.0.6b73d90.dist-info/top_level.txt'
  adding 'opencv_python_headless-4.7.0.6b73d90.dist-info/RECORD'
  removing _skbuild/linux-x86_64-3.8/setuptools/bdist.linux-x86_64/wheel
  /tmp/pip-build-env-2l3nyybo/overlay/lib/python3.8/site-packages/setuptools/dist.py:501: UserWarning: The version specified ('4.7.0.6b73d90') is an invalid version, this may not work as expected with newer versions of setuptools, pip, and PyPI. Please see PEP 440 for more details.
    warnings.warn(
  /tmp/pip-build-env-2l3nyybo/overlay/lib/python3.8/site-packages/setuptools/dist.py:501: UserWarning: The version specified ('4.7.0.6b73d90') is an invalid version, this may not work as expected with newer versions of setuptools, pip, and PyPI. Please see PEP 440 for more details.
    warnings.warn(
  /tmp/pip-build-env-2l3nyybo/overlay/lib/python3.8/site-packages/pkg_resources/__init__.py:116: PkgResourcesDeprecationWarning: 4.7.0.6b73d90 is an invalid version and will not be supported in a future release
    warnings.warn(
  Building wheel for opencv-python-headless (pyproject.toml) ... done
  Created wheel for opencv-python-headless: filename=opencv_python_headless-4.7.0.6b73d90-cp38-cp38-linux_x86_64.whl size=28777931 sha256=c23a3695c39ffb7b07dbae430ef3145f834e213b5fe0e6c06c124bd275f83cb3
  Stored in directory: /tmp/pip-ephem-wheel-cache-w0lpva8d/wheels/8d/df/01/3502823b3430c611058d0842c6059ecd1c1ef33537da86e988
  WARNING: Built wheel for opencv-python-headless is invalid: Metadata 1.2 mandates PEP 440 version, but '4.7.0.6b73d90' is not
Failed to build opencv-python-headless
ERROR: Failed to build one or more wheels
```

</p></details>

(Note: I had to cut off several long passages due to GH's 64k char size constraint, each marked as `...`.)

I can only see two potential problems:
1. it seems like it's trying to use python2.7 at times (see `Traceback` and `distutils-precedence`.)
2. it complains `Built wheel for opencv-python-headless is invalid: Metadata 1.2 mandates PEP 440 version, but '4.7.0.6b73d90' is not`

However, 1. seems to be ignored and later OpenCV gets built and the Python wheels (temporarily), too.

And regarding 2., this is only a warning message. (And I don't see why I must conform to that naming scheme for a local distribution. Furthermore, at least for the CI build, it seems ok to build from a non-tagged commit, as [this successful CI log for the same commit](https://github.com/opencv/opencv-python/actions/runs/4705682937/jobs/8346401900) shows.)

### Steps to reproduce

I followed the [instructions for manual builds](https://github.com/opencv/opencv-python#manual-builds), except that in my case opencv-python is a submodule, so it's `git submodule update --init --recursive  opencv-python` instead of `git clone --recursive https://github.com/opencv/opencv-python.git`.

<details><summary>output of git clone</summary>
<p>

```
Submodule path 'opencv-python': checked out '6b73d90fc3e50ba6858926d299b49f0228e19d68'
Submodule path 'opencv-python/multibuild': checked out '15fd8770abe6b12e94c2362ae8f21cd876bd998e'
Submodule path 'opencv-python/opencv': checked out '725e440d278aca07d35a5e8963ef990572b07316'
Submodule path 'opencv-python/opencv_contrib': checked out 'e247b680a6bd396f110274b6c214406a93171350'
Submodule path 'opencv-python/opencv_extra': checked out '5abbd7e0546bbb34ae7487170383d3e571fb1dd1'
Submodule 'opencv-python' (https://github.com/skvark/opencv-python.git) registered for path 'opencv-python'
Cloning into '/test/opencv-python'...
Submodule 'multibuild' (https://github.com/multi-build/multibuild.git) registered for path 'opencv-python/multibuild'
Submodule 'opencv' (https://github.com/opencv/opencv.git) registered for path 'opencv-python/opencv'
Submodule 'opencv_contrib' (https://github.com/opencv/opencv_contrib.git) registered for path 'opencv-python/opencv_contrib'
Submodule 'opencv_extra' (https://github.com/opencv/opencv_extra.git) registered for path 'opencv-python/opencv_extra'
Cloning into '/test/opencv-python/multibuild'...
Cloning into '/test/opencv-python/opencv'...
Cloning into '/test/opencv-python/opencv_contrib'...
Cloning into '/test/opencv-python/opencv_extra'...
```

</p></details>

Commands to build:

    python3 -m venv /test/venv
    . /test/venv/bin/activate && pip install --upgrade pip setuptools wheel
    . /test/venv/bin/activate && ENABLE_HEADLESS=1 pip wheel ./opencv-python --verbose

My environment is:
- Ubuntu 20.04 distribution of GNU/Linux 5.4.0 on x86_64
- Python 3.8.10 / Pip 23.1
- opencv-python 6b73d90fc3e50ba6858926d299b49f0228e19d68 (current head of 4.x)

##### Issue submission checklist

 - [x] This is not a generic OpenCV usage question (looking for help for coding, other usage questions, homework etc.)
 - [x] I have read the README of this repository and understand that this repository provides only an automated build toolchain for OpenCV Python packages (there is no actual OpenCV code here)
 - [x] The issue is related to the build scripts in this repository, to the pre-built binaries or is a feature request (such as "please enable this additional dependency")
 - [x] I'm using the latest version of ``opencv-python``

Update: found a [thread on the OpenCV forum](https://forum.opencv.org/t/error-building-wheel-for-opencv-python-from-source/9850) about this exact problem. The OP there was indeed able to get it done by downgrading to a tagged commit.

So I tried that with f9de34ec8bc8bc5d61e5e35c0721676993da7d71 (which is tagged `72` on 4.x), but then I get another error:

<details><summary>build log</summary>
<p>

```
Looking in indexes: https://pypi.org/simple, https://pypi.ngc.nvidia.com
Processing ./opencv-python
  Running command pip subprocess to install build dependencies
  Looking in indexes: https://pypi.org/simple, https://pypi.ngc.nvidia.com, https://pypi.ngc.nvidia.com
  Ignoring numpy: markers 'python_version == "3.6" and platform_machine != "aarch64" and platform_machine != "arm64"' don't match your environment
  Ignoring numpy: markers 'python_version == "3.7" and platform_machine != "aarch64" and platform_machine != "arm64"' don't match your environment
  Ignoring numpy: markers 'python_version <= "3.9" and sys_platform == "linux" and platform_machine == "aarch64"' don't match your environment
  Ignoring numpy: markers 'python_version <= "3.9" and sys_platform == "darwin" and platform_machine == "arm64"' don't match your environment
  Ignoring numpy: markers 'python_version == "3.9" and platform_machine != "aarch64" and platform_machine != "arm64"' don't match your environment
  Ignoring numpy: markers 'python_version == "3.10" and platform_system != "Darwin"' don't match your environment
  Ignoring numpy: markers 'python_version == "3.10" and platform_system == "Darwin"' don't match your environment
  Ignoring numpy: markers 'python_version >= "3.11"' don't match your environment
  Collecting setuptools==59.2.0
    Downloading setuptools-59.2.0-py3-none-any.whl (952 kB)
       ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 952.0/952.0 kB 15.8 MB/s eta 0:00:00
  Collecting wheel==0.37.0
    Downloading wheel-0.37.0-py2.py3-none-any.whl (35 kB)
  Collecting cmake>=3.1
    Downloading cmake-3.26.3-py2.py3-none-manylinux2014_x86_64.manylinux_2_17_x86_64.whl (24.0 MB)
       ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 24.0/24.0 MB 96.8 MB/s eta 0:00:00
  Collecting pip
    Downloading pip-23.1.1-py3-none-any.whl (2.1 MB)
       ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 2.1/2.1 MB 107.5 MB/s eta 0:00:00
  Collecting scikit-build>=0.13.2
    Downloading scikit_build-0.17.2-py3-none-any.whl (82 kB)
       ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 82.1/82.1 kB 332.6 MB/s eta 0:00:00
  Collecting numpy==1.17.3
    Downloading numpy-1.17.3-cp38-cp38-manylinux1_x86_64.whl (20.5 MB)
       ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 20.5/20.5 MB 83.4 MB/s eta 0:00:00
  Collecting distro (from scikit-build>=0.13.2)
    Downloading distro-1.8.0-py3-none-any.whl (20 kB)
  Collecting packaging (from scikit-build>=0.13.2)
    Downloading packaging-23.1-py3-none-any.whl (48 kB)
       ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 48.9/48.9 kB 297.0 MB/s eta 0:00:00
  Collecting tomli (from scikit-build>=0.13.2)
    Downloading tomli-2.0.1-py3-none-any.whl (12 kB)
  Installing collected packages: cmake, wheel, tomli, setuptools, pip, packaging, numpy, distro, scikit-build
  Successfully installed cmake-3.26.3 distro-1.8.0 numpy-1.17.3 packaging-23.1 pip-23.1.1 scikit-build-0.17.2 setuptools-59.2.0 tomli-2.0.1 wheel-0.37.0
  Installing build dependencies ... done
  Running command Getting requirements to build wheel
  running egg_info
  writing opencv_python_headless.egg-info/PKG-INFO
  writing dependency_links to opencv_python_headless.egg-info/dependency_links.txt
  writing requirements to opencv_python_headless.egg-info/requires.txt
  writing top-level names to opencv_python_headless.egg-info/top_level.txt
  reading manifest file 'opencv_python_headless.egg-info/SOURCES.txt'
  reading manifest template 'MANIFEST.in'
  adding license file 'LICENSE-3RD-PARTY.txt'
  adding license file 'LICENSE.txt'
  writing manifest file 'opencv_python_headless.egg-info/SOURCES.txt'
  Getting requirements to build wheel ... done
  Running command Preparing metadata (pyproject.toml)
  running dist_info
  creating /tmp/pip-modern-metadata-4wjs61vd/opencv_python_headless.egg-info
  writing /tmp/pip-modern-metadata-4wjs61vd/opencv_python_headless.egg-info/PKG-INFO
  writing dependency_links to /tmp/pip-modern-metadata-4wjs61vd/opencv_python_headless.egg-info/dependency_links.txt
  writing requirements to /tmp/pip-modern-metadata-4wjs61vd/opencv_python_headless.egg-info/requires.txt
  writing top-level names to /tmp/pip-modern-metadata-4wjs61vd/opencv_python_headless.egg-info/top_level.txt
  writing manifest file '/tmp/pip-modern-metadata-4wjs61vd/opencv_python_headless.egg-info/SOURCES.txt'
  reading manifest file '/tmp/pip-modern-metadata-4wjs61vd/opencv_python_headless.egg-info/SOURCES.txt'
  reading manifest template 'MANIFEST.in'
  adding license file 'LICENSE-3RD-PARTY.txt'
  adding license file 'LICENSE.txt'
  writing manifest file '/tmp/pip-modern-metadata-4wjs61vd/opencv_python_headless.egg-info/SOURCES.txt'
  creating '/tmp/pip-modern-metadata-4wjs61vd/opencv_python_headless.dist-info'
  adding license file "LICENSE.txt" (matched pattern "LICEN[CS]E*")
  adding license file "LICENSE-3RD-PARTY.txt" (matched pattern "LICEN[CS]E*")
  Preparing metadata (pyproject.toml) ... done
Collecting numpy>=1.17.0 (from opencv-python-headless==4.7.0.72)
  File was already downloaded /test/numpy-1.24.3-cp38-cp38-manylinux_2_17_x86_64.manylinux2014_x86_64.whl
Building wheels for collected packages: opencv-python-headless
  Running command Building wheel for opencv-python-headless (pyproject.toml)


  --------------------------------------------------------------------------------
  -- Trying 'Ninja' generator
  --------------------------------
  ---------------------------
  ----------------------
  -----------------
  ------------
  -------
  --
  CMake Error: CMake was unable to find a build program corresponding to "Ninja".  CMAKE_MAKE_PROGRAM is not set.  You probably need to select a different build tool.Not searching for unused variables given on the command line.

  -- Configuring incomplete, errors occurred!
  --
  -------
  ------------
  -----------------
  ----------------------
  ---------------------------
  --------------------------------
  -- Trying 'Ninja' generator - failure
  --------------------------------------------------------------------------------



  --------------------------------------------------------------------------------
  -- Trying 'Unix Makefiles' generator
  --------------------------------
  ---------------------------
  ----------------------
  -----------------
  ------------
  -------
  --
  Not searching for unused variables given on the command line.
  -- The C compiler identification is GNU 9.4.0
  -- Detecting C compiler ABI info
  -- Detecting C compiler ABI info - done
  -- Check for working C compiler: /usr/bin/cc - skipped
  -- Detecting C compile features
  -- Detecting C compile features - done
  -- The CXX compiler identification is GNU 9.4.0
  -- Detecting CXX compiler ABI info
  -- Detecting CXX compiler ABI info - done
  -- Check for working CXX compiler: /usr/bin/c++ - skipped
  -- Detecting CXX compile features
  -- Detecting CXX compile features - done
  -- Configuring done (0.2s)
  -- Generating done (0.0s)
  -- Build files have been written to: /test/opencv-python/_cmake_test_compile/build
  --
  -------
  ------------
  -----------------
  ----------------------
  ---------------------------
  --------------------------------
  -- Trying 'Unix Makefiles' generator - success
  --------------------------------------------------------------------------------

  Configuring Project
    Working directory:
      /test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build
    Command:
      /tmp/pip-build-env-rmlqcdjw/overlay/lib/python3.8/site-packages/cmake/data/bin/cmake /test/opencv-python/opencv -G 'Unix Makefiles' --no-warn-unused-cli -DCMAKE_INSTALL_PREFIX:PATH=/test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-install -DPYTHON_VERSION_STRING:STRING=3.8.10 -DSKBUILD:INTERNAL=TRUE -DCMAKE_MODULE_PATH:PATH=/tmp/pip-build-env-rmlqcdjw/overlay/lib/python3.8/site-packages/skbuild/resources/cmake -DPYTHON_EXECUTABLE:PATH=/test/venv/bin/python3 -DPYTHON_INCLUDE_DIR:PATH=/usr/include/python3.8 -DPYTHON_LIBRARY:PATH=/usr/lib/x86_64-linux-gnu/libpython3.8.so -DPython_EXECUTABLE:PATH=/test/venv/bin/python3 -DPython_ROOT_DIR:PATH=/test/venv -DPython_FIND_REGISTRY:STRING=NEVER -DPython_INCLUDE_DIR:PATH=/usr/include/python3.8 -DPython_NumPy_INCLUDE_DIRS:PATH=/tmp/pip-build-env-rmlqcdjw/overlay/lib/python3.8/site-packages/numpy/core/include -DPython3_EXECUTABLE:PATH=/test/venv/bin/python3 -DPython3_ROOT_DIR:PATH=/test/venv -DPython3_FIND_REGISTRY:STRING=NEVER -DPython3_INCLUDE_DIR:PATH=/usr/include/python3.8 -DPython3_NumPy_INCLUDE_DIRS:PATH=/tmp/pip-build-env-rmlqcdjw/overlay/lib/python3.8/site-packages/numpy/core/include -DPYTHON3_EXECUTABLE=/test/venv/bin/python3 -DPYTHON3_INCLUDE_DIR=/usr/include/python3.8 -DPYTHON3_LIBRARY=/usr/lib/x86_64-linux-gnu/libpython3.8.so -DBUILD_opencv_python3=ON -DBUILD_opencv_python2=OFF -DBUILD_opencv_java=OFF -DOPENCV_PYTHON3_INSTALL_PATH=python -DINSTALL_CREATE_DISTRIB=ON -DBUILD_opencv_apps=OFF -DBUILD_opencv_freetype=OFF -DBUILD_SHARED_LIBS=OFF -DBUILD_TESTS=OFF -DBUILD_PERF_TESTS=OFF -DBUILD_DOCS=OFF -DPYTHON3_LIMITED_API=ON -DBUILD_OPENEXR=ON -DWITH_WIN32UI=OFF -DWITH_QT=OFF -DWITH_GTK=OFF -DCMAKE_BUILD_TYPE:STRING=Release

  Not searching for unused variables given on the command line.
  -- Detected processor: x86_64
  -- Looking for ccache - not found
  -- Found ZLIB: /usr/lib/x86_64-linux-gnu/libz.so (found suitable version "1.2.11", minimum required is "1.2.3")
  -- Could NOT find OpenJPEG (minimal suitable version: 2.0, recommended version >= 2.3.1). OpenJPEG will be built from sources
  -- OpenJPEG: VERSION = 2.4.0, BUILD = opencv-4.7.0-openjp2-2.4.0
  -- OpenJPEG libraries will be built from sources: libopenjp2 (version "2.4.0")
  -- Found ZLIB: /usr/lib/x86_64-linux-gnu/libz.so (found version "1.2.11")
  -- libva: missing va.h header (VA_INCLUDE_DIR)
  -- found Intel IPP (ICV version): 2020.0.0 [2020.0.0 Gold]
  -- at: /test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build/3rdparty/ippicv/ippicv_lnx/icv
  -- found Intel IPP Integration Wrappers sources: 2020.0.0
  -- at: /test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build/3rdparty/ippicv/ippicv_lnx/iw
  -- Could not find OpenBLAS include. Turning OpenBLAS_FOUND off
  -- Could not find OpenBLAS lib. Turning OpenBLAS_FOUND off
  -- Could NOT find Atlas (missing: Atlas_CBLAS_INCLUDE_DIR Atlas_CLAPACK_INCLUDE_DIR Atlas_CBLAS_LIBRARY Atlas_BLAS_LIBRARY Atlas_LAPACK_LIBRARY)
  -- Could NOT find BLAS (missing: BLAS_LIBRARIES)
  -- Could NOT find LAPACK (missing: LAPACK_LIBRARIES)
      Reason given by package: LAPACK could not be found because dependency BLAS could not be found.

  -- Could NOT find JNI (missing: JAVA_INCLUDE_PATH JAVA_INCLUDE_PATH2 AWT)
  -- VTK is not found. Please set -DVTK_DIR in CMake to VTK build directory, or to VTK install subdirectory with VTKConfig.cmake file
  -- Checking for modules 'libavcodec;libavformat;libavutil;libswscale'
  --   No package 'libavcodec' found
  --   No package 'libavformat' found
  --   No package 'libavutil' found
  --   No package 'libswscale' found
  -- FFMPEG is disabled. Required libraries: libavcodec;libavformat;libavutil;libswscale. Missing libraries: libavcodec;libavformat;libavutil;libswscale
  -- Checking for module 'gstreamer-base-1.0'
  --   No package 'gstreamer-base-1.0' found
  -- Checking for module 'gstreamer-app-1.0'
  --   No package 'gstreamer-app-1.0' found
  -- Checking for module 'gstreamer-riff-1.0'
  --   No package 'gstreamer-riff-1.0' found
  -- Checking for module 'gstreamer-pbutils-1.0'
  --   No package 'gstreamer-pbutils-1.0' found
  -- Checking for module 'gstreamer-video-1.0'
  --   No package 'gstreamer-video-1.0' found
  -- Checking for module 'gstreamer-audio-1.0'
  --   No package 'gstreamer-audio-1.0' found
  -- Checking for module 'libdc1394-2'
  --   No package 'libdc1394-2' found
  -- Allocator metrics storage type: 'long long'
  -- Excluding from source files list: modules/imgproc/src/imgwarp.lasx.cpp
  -- Excluding from source files list: modules/imgproc/src/resize.lasx.cpp
  -- Registering hook 'INIT_MODULE_SOURCES_opencv_dnn': /test/opencv-python/opencv/modules/dnn/cmake/hooks/INIT_MODULE_SOURCES_opencv_dnn.cmake
  -- opencv_dnn: filter out cuda4dnn source code
  -- Excluding from source files list: <BUILD>/modules/dnn/layers/layers_common.rvv.cpp
  -- Excluding from source files list: <BUILD>/modules/dnn/layers/layers_common.lasx.cpp
  -- Excluding from source files list: <BUILD>/modules/dnn/int8layers/layers_common.lasx.cpp
  -- imgcodecs: OpenEXR codec is disabled in runtime. Details: https://github.com/opencv/opencv/issues/21326
  -- highgui: using builtin backend: NONE
  -- Found 'misc' Python modules from /test/opencv-python/opencv/modules/python/package/extra_modules
  -- Found 'mat_wrapper;utils' Python modules from /test/opencv-python/opencv/modules/core/misc/python/package
  -- Found 'gapi' Python modules from /test/opencv-python/opencv/modules/gapi/misc/python/package
  -- Found 'misc' Python modules from /test/opencv-python/opencv/modules/python/package/extra_modules
  -- Found 'mat_wrapper;utils' Python modules from /test/opencv-python/opencv/modules/core/misc/python/package
  -- Found 'gapi' Python modules from /test/opencv-python/opencv/modules/gapi/misc/python/package
  --
  -- General configuration for OpenCV 4.7.0 =====================================
  --   Version control:               4.7.0
  --
  --   Platform:
  --     Timestamp:                   2023-04-25T16:33:28Z
  --     Host:                        Linux 5.4.0-147-generic x86_64
  --     CMake:                       3.26.3
  --     CMake generator:             Unix Makefiles
  --     CMake build tool:            /usr/bin/make
  --     Configuration:               Release
  --
  --   CPU/HW features:
  --     Baseline:                    SSE SSE2 SSE3
  --       requested:                 SSE3
  --     Dispatched code generation:  SSE4_1 SSE4_2 FP16 AVX AVX2 AVX512_SKX
  --       requested:                 SSE4_1 SSE4_2 AVX FP16 AVX2 AVX512_SKX
  --       SSE4_1 (16 files):         + SSSE3 SSE4_1
  --       SSE4_2 (1 files):          + SSSE3 SSE4_1 POPCNT SSE4_2
  --       FP16 (0 files):            + SSSE3 SSE4_1 POPCNT SSE4_2 FP16 AVX
  --       AVX (4 files):             + SSSE3 SSE4_1 POPCNT SSE4_2 AVX
  --       AVX2 (32 files):           + SSSE3 SSE4_1 POPCNT SSE4_2 FP16 FMA3 AVX AVX2
  --       AVX512_SKX (5 files):      + SSSE3 SSE4_1 POPCNT SSE4_2 FP16 FMA3 AVX AVX2 AVX_512F AVX512_COMMON AVX512_SKX
  --
  --   C/C++:
  --     Built as dynamic libs?:      NO
  --     C++ standard:                11
  --     C++ Compiler:                /usr/bin/c++  (ver 9.4.0)
  --     C++ flags (Release):         -fsigned-char -W -Wall -Wreturn-type -Wnon-virtual-dtor -Waddress -Wsequence-point -Wformat -Wformat-security -Wmissing-declarations -Wundef -Winit-self -Wpointer-arith -Wshadow -Wsign-promo -Wuninitialized -Wsuggest-override -Wno-delete-non-virtual-dtor -Wno-comment -Wimplicit-fallthrough=3 -Wno-strict-overflow -fdiagnostics-show-option -Wno-long-long -pthread -fomit-frame-pointer -ffunction-sections -fdata-sections  -msse -msse2 -msse3 -fvisibility=hidden -fvisibility-inlines-hidden -O3 -DNDEBUG  -DNDEBUG
  --     C++ flags (Debug):           -fsigned-char -W -Wall -Wreturn-type -Wnon-virtual-dtor -Waddress -Wsequence-point -Wformat -Wformat-security -Wmissing-declarations -Wundef -Winit-self -Wpointer-arith -Wshadow -Wsign-promo -Wuninitialized -Wsuggest-override -Wno-delete-non-virtual-dtor -Wno-comment -Wimplicit-fallthrough=3 -Wno-strict-overflow -fdiagnostics-show-option -Wno-long-long -pthread -fomit-frame-pointer -ffunction-sections -fdata-sections  -msse -msse2 -msse3 -fvisibility=hidden -fvisibility-inlines-hidden -g  -O0 -DDEBUG -D_DEBUG
  --     C Compiler:                  /usr/bin/cc
  --     C flags (Release):           -fsigned-char -W -Wall -Wreturn-type -Waddress -Wsequence-point -Wformat -Wformat-security -Wmissing-declarations -Wmissing-prototypes -Wstrict-prototypes -Wundef -Winit-self -Wpointer-arith -Wshadow -Wuninitialized -Wno-comment -Wimplicit-fallthrough=3 -Wno-strict-overflow -fdiagnostics-show-option -Wno-long-long -pthread -fomit-frame-pointer -ffunction-sections -fdata-sections  -msse -msse2 -msse3 -fvisibility=hidden -O3 -DNDEBUG  -DNDEBUG
  --     C flags (Debug):             -fsigned-char -W -Wall -Wreturn-type -Waddress -Wsequence-point -Wformat -Wformat-security -Wmissing-declarations -Wmissing-prototypes -Wstrict-prototypes -Wundef -Winit-self -Wpointer-arith -Wshadow -Wuninitialized -Wno-comment -Wimplicit-fallthrough=3 -Wno-strict-overflow -fdiagnostics-show-option -Wno-long-long -pthread -fomit-frame-pointer -ffunction-sections -fdata-sections  -msse -msse2 -msse3 -fvisibility=hidden -g  -O0 -DDEBUG -D_DEBUG
  --     Linker flags (Release):      -Wl,--exclude-libs,libippicv.a -Wl,--exclude-libs,libippiw.a   -Wl,--gc-sections -Wl,--as-needed -Wl,--no-undefined
  --     Linker flags (Debug):        -Wl,--exclude-libs,libippicv.a -Wl,--exclude-libs,libippiw.a   -Wl,--gc-sections -Wl,--as-needed -Wl,--no-undefined
  --     ccache:                      NO
  --     Precompiled headers:         NO
  --     Extra dependencies:          /usr/lib/x86_64-linux-gnu/libjpeg.so /usr/lib/x86_64-linux-gnu/libwebp.so /usr/lib/x86_64-linux-gnu/libpng.so /usr/lib/x86_64-linux-gnu/libtiff.so /usr/lib/x86_64-linux-gnu/libz.so dl m pthread rt
  --     3rdparty dependencies:       libprotobuf ade ittnotify libopenjp2 IlmImf quirc ippiw ippicv
  --
  --   OpenCV modules:
  --     To be built:                 calib3d core dnn features2d flann gapi highgui imgcodecs imgproc ml objdetect photo python3 stitching video videoio
  --     Disabled:                    world
  --     Disabled by dependency:      -
  --     Unavailable:                 java python2 ts
  --     Applications:                -
  --     Documentation:               NO
  --     Non-free algorithms:         NO
  --
  --   GUI:                           NONE
  --     VTK support:                 NO
  --
  --   Media I/O:
  --     ZLib:                        /usr/lib/x86_64-linux-gnu/libz.so (ver 1.2.11)
  --     JPEG:                        /usr/lib/x86_64-linux-gnu/libjpeg.so (ver 80)
  --     WEBP:                        /usr/lib/x86_64-linux-gnu/libwebp.so (ver encoder: 0x020e)
  --     PNG:                         /usr/lib/x86_64-linux-gnu/libpng.so (ver 1.6.37)
  --     TIFF:                        /usr/lib/x86_64-linux-gnu/libtiff.so (ver 42 / 4.1.0)
  --     JPEG 2000:                   build (ver 2.4.0)
  --     OpenEXR:                     build (ver 2.3.0)
  --     HDR:                         YES
  --     SUNRASTER:                   YES
  --     PXM:                         YES
  --     PFM:                         YES
  --
  --   Video I/O:
  --     DC1394:                      NO
  --     FFMPEG:                      NO
  --       avcodec:                   NO
  --       avformat:                  NO
  --       avutil:                    NO
  --       swscale:                   NO
  --       avresample:                NO
  --     GStreamer:                   NO
  --     v4l/v4l2:                    YES (linux/videodev2.h)
  --
  --   Parallel framework:            pthreads
  --
  --   Trace:                         YES (with Intel ITT)
  --
  --   Other third-party libraries:
  --     Intel IPP:                   2020.0.0 Gold [2020.0.0]
  --            at:                   /test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build/3rdparty/ippicv/ippicv_lnx/icv
  --     Intel IPP IW:                sources (2020.0.0)
  --               at:                /test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build/3rdparty/ippicv/ippicv_lnx/iw
  --     VA:                          NO
  --     Lapack:                      NO
  --     Eigen:                       NO
  --     Custom HAL:                  NO
  --     Protobuf:                    build (3.19.1)
  --
  --   OpenCL:                        YES (no extra features)
  --     Include path:                /test/opencv-python/opencv/3rdparty/include/opencl/1.2
  --     Link libraries:              Dynamic load
  --
  --   Python 3:
  --     Interpreter:                 /test/venv/bin/python3 (ver 3.8.10)
  --     Libraries:                   /usr/lib/x86_64-linux-gnu/libpython3.8.so (ver 3.8.10)
  --     numpy:                       /tmp/pip-build-env-2l3nyybo/overlay/lib/python3.8/site-packages/numpy/core/include (ver 1.17.3)
  --     install path:                python/cv2/python-3
  --
  --   Python (for build):            /usr/bin/python2.7
  --
  --   Java:
  --     ant:                         NO
  --     JNI:                         NO
  --     Java wrappers:               NO
  --     Java tests:                  NO
  --
  --   Install to:                    /test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-install
  -- -----------------------------------------------------------------
  --
  -- Configuring done (0.7s)
  -- Generating done (0.2s)
  -- Build files have been written to: /test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build
  make[1]: Entering directory '/test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build'
  make[2]: Entering directory '/test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build'
  make[3]: Entering directory '/test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build'
  make[3]: Leaving directory '/test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build'
  [  0%] Built target opencv_dnn_plugins
  make[3]: Entering directory '/test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build'
  make[3]: Leaving directory '/test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build'
  [  0%] Built target opencv_highgui_plugins
  make[3]: Entering directory '/test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build'
  make[3]: Leaving directory '/test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build'
  make[3]: Entering directory '/test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build'
  [  0%] Linking C static library ../../lib/liblibopenjp2.a
  make[3]: Leaving directory '/test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build'
  [  2%] Built target libopenjp2
  make[3]: Entering directory '/test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build'
  make[3]: Leaving directory '/test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build'
  [  2%] Built target opencv_videoio_plugins
  make[3]: Entering directory '/test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build'
  make[3]: Leaving directory '/test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build'
  make[3]: Entering directory '/test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build'
  [  2%] Linking C static library ../lib/libittnotify.a
  make[3]: Leaving directory '/test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build'
  [  2%] Built target ittnotify
  make[3]: Entering directory '/test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build'
  make[3]: Leaving directory '/test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build'
  make[3]: Entering directory '/test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build'
  [  3%] Linking C static library ../lib/libippiw.a
  make[3]: Leaving directory '/test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build'
  [  5%] Built target ippiw
  make[3]: Entering directory '/test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build'
  [  5%] Processing OpenCL kernels (core)
  -- /test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build/modules/core/opencl_kernels_core.hpp contains the same content
  make[3]: Leaving directory '/test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build'
  make[3]: Entering directory '/test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build'
  [  6%] Building CXX object modules/core/CMakeFiles/opencv_core.dir/opencl_kernels_core.cpp.o
  [  6%] Linking CXX static library ../../lib/libopencv_core.a
  make[3]: Leaving directory '/test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build'
  [ 16%] Built target opencv_core
  make[3]: Entering directory '/test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build'
  make[3]: Leaving directory '/test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build'
  make[3]: Entering directory '/test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build'
  [ 16%] Linking CXX static library ../../lib/libopencv_flann.a
  make[3]: Leaving directory '/test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build'
  [ 17%] Built target opencv_flann
  make[3]: Entering directory '/test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build'
  [ 17%] Processing OpenCL kernels (imgproc)
  -- /test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build/modules/imgproc/opencl_kernels_imgproc.hpp contains the same content
  make[3]: Leaving directory '/test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build'
  make[3]: Entering directory '/test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build'
  [ 17%] Building CXX object modules/imgproc/CMakeFiles/opencv_imgproc.dir/opencl_kernels_imgproc.cpp.o
  [ 17%] Linking CXX static library ../../lib/libopencv_imgproc.a
  make[3]: Leaving directory '/test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build'
  [ 27%] Built target opencv_imgproc
  make[3]: Entering directory '/test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build'
  make[3]: Leaving directory '/test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build'
  make[3]: Entering directory '/test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build'
  [ 28%] Linking CXX static library ../../lib/libopencv_ml.a
  make[3]: Leaving directory '/test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build'
  [ 29%] Built target opencv_ml
  make[3]: Entering directory '/test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build'
  [ 29%] Processing OpenCL kernels (photo)
  -- /test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build/modules/photo/opencl_kernels_photo.hpp contains the same content
  make[3]: Leaving directory '/test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build'
  make[3]: Entering directory '/test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build'
  [ 29%] Building CXX object modules/photo/CMakeFiles/opencv_photo.dir/opencl_kernels_photo.cpp.o
  [ 29%] Linking CXX static library ../../lib/libopencv_photo.a
  make[3]: Leaving directory '/test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build'
  [ 30%] Built target opencv_photo
  make[3]: Entering directory '/test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build'
  make[3]: Leaving directory '/test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build'
  make[3]: Entering directory '/test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build'
  [ 30%] Linking CXX static library ../lib/liblibprotobuf.a
  make[3]: Leaving directory '/test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build'
  [ 35%] Built target libprotobuf
  make[3]: Entering directory '/test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build'
  [ 36%] Processing OpenCL kernels (dnn)
  -- /test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build/modules/dnn/opencl_kernels_dnn.hpp contains the same content
  make[3]: Leaving directory '/test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build'
  make[3]: Entering directory '/test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build'
  [ 36%] Building CXX object modules/dnn/CMakeFiles/opencv_dnn.dir/opencl_kernels_dnn.cpp.o
  [ 36%] Linking CXX static library ../../lib/libopencv_dnn.a
  make[3]: Leaving directory '/test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build'
  [ 52%] Built target opencv_dnn
  make[3]: Entering directory '/test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build'
  [ 52%] Processing OpenCL kernels (features2d)
  -- /test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build/modules/features2d/opencl_kernels_features2d.hpp contains the same content
  make[3]: Leaving directory '/test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build'
  make[3]: Entering directory '/test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build'
  [ 52%] Building CXX object modules/features2d/CMakeFiles/opencv_features2d.dir/opencl_kernels_features2d.cpp.o
  [ 52%] Linking CXX static library ../../lib/libopencv_features2d.a
  make[3]: Leaving directory '/test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build'
  [ 55%] Built target opencv_features2d
  make[3]: Entering directory '/test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build'
  make[3]: Leaving directory '/test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build'
  make[3]: Entering directory '/test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build'
  [ 55%] Linking CXX static library ../lib/libIlmImf.a
  make[3]: Leaving directory '/test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build'
  [ 66%] Built target IlmImf
  make[3]: Entering directory '/test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build'
  make[3]: Leaving directory '/test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build'
  make[3]: Entering directory '/test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build'
  [ 66%] Linking CXX static library ../../lib/libopencv_imgcodecs.a
  make[3]: Leaving directory '/test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build'
  [ 68%] Built target opencv_imgcodecs
  make[3]: Entering directory '/test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build'
  make[3]: Leaving directory '/test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build'
  make[3]: Entering directory '/test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build'
  [ 68%] Linking CXX static library ../../lib/libopencv_videoio.a
  make[3]: Leaving directory '/test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build'
  [ 69%] Built target opencv_videoio
  make[3]: Entering directory '/test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build'
  [ 69%] Processing OpenCL kernels (calib3d)
  -- /test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build/modules/calib3d/opencl_kernels_calib3d.hpp contains the same content
  make[3]: Leaving directory '/test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build'
  make[3]: Entering directory '/test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build'
  [ 69%] Building CXX object modules/calib3d/CMakeFiles/opencv_calib3d.dir/opencl_kernels_calib3d.cpp.o
  [ 69%] Linking CXX static library ../../lib/libopencv_calib3d.a
  make[3]: Leaving directory '/test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build'
  [ 74%] Built target opencv_calib3d
  make[3]: Entering directory '/test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build'
  make[3]: Leaving directory '/test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build'
  make[3]: Entering directory '/test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build'
  [ 74%] Linking CXX static library ../../lib/libopencv_highgui.a
  make[3]: Leaving directory '/test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build'
  [ 75%] Built target opencv_highgui
  make[3]: Entering directory '/test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build'
  make[3]: Leaving directory '/test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build'
  make[3]: Entering directory '/test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build'
  [ 76%] Linking C static library ../lib/libquirc.a
  make[3]: Leaving directory '/test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build'
  [ 76%] Built target quirc
  make[3]: Entering directory '/test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build'
  [ 76%] Processing OpenCL kernels (objdetect)
  -- /test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build/modules/objdetect/opencl_kernels_objdetect.hpp contains the same content
  make[3]: Leaving directory '/test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build'
  make[3]: Entering directory '/test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build'
  [ 77%] Building CXX object modules/objdetect/CMakeFiles/opencv_objdetect.dir/opencl_kernels_objdetect.cpp.o
  [ 77%] Linking CXX static library ../../lib/libopencv_objdetect.a
  make[3]: Leaving directory '/test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build'
  [ 78%] Built target opencv_objdetect
  make[3]: Entering directory '/test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build'
  [ 78%] Processing OpenCL kernels (stitching)
  -- /test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build/modules/stitching/opencl_kernels_stitching.hpp contains the same content
  make[3]: Leaving directory '/test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build'
  make[3]: Entering directory '/test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build'
  [ 78%] Building CXX object modules/stitching/CMakeFiles/opencv_stitching.dir/opencl_kernels_stitching.cpp.o
  [ 79%] Linking CXX static library ../../lib/libopencv_stitching.a
  make[3]: Leaving directory '/test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build'
  [ 80%] Built target opencv_stitching
  make[3]: Entering directory '/test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build'
  [ 80%] Processing OpenCL kernels (video)
  -- /test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build/modules/video/opencl_kernels_video.hpp contains the same content
  make[3]: Leaving directory '/test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build'
  make[3]: Entering directory '/test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build'
  [ 80%] Building CXX object modules/video/CMakeFiles/opencv_video.dir/opencl_kernels_video.cpp.o
  [ 81%] Linking CXX static library ../../lib/libopencv_video.a
  make[3]: Leaving directory '/test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build'
  [ 83%] Built target opencv_video
  make[3]: Entering directory '/test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build'
  make[3]: Leaving directory '/test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build'
  make[3]: Entering directory '/test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build'
  [ 83%] Linking CXX static library 3rdparty/lib/libade.a
  make[3]: Leaving directory '/test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build'
  [ 85%] Built target ade
  make[3]: Entering directory '/test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build'
  make[3]: Leaving directory '/test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build'
  make[3]: Entering directory '/test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build'
  [ 85%] Linking CXX static library ../../lib/libopencv_gapi.a
  make[3]: Leaving directory '/test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build'
  [ 98%] Built target opencv_gapi
  make[3]: Entering directory '/test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build'
  make[3]: Leaving directory '/test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build'
  [ 99%] Built target gen_opencv_python_source
  make[3]: Entering directory '/test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build'
  make[3]: Leaving directory '/test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build'
  make[3]: Entering directory '/test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build'
  [ 99%] Building CXX object modules/python3/CMakeFiles/opencv_python3.dir/__/src2/cv2.cpp.o
  In file included from /test/opencv-python/opencv/modules/python/src2/cv2.cpp:5:
  /test/opencv-python/opencv/modules/python/src2/cv2.hpp:36:10: fatal error: numpy/ndarrayobject.h: No such file or directory
     36 | #include <numpy/ndarrayobject.h>
        |          ^~~~~~~~~~~~~~~~~~~~~~~
  compilation terminated.
  make[3]: *** [modules/python3/CMakeFiles/opencv_python3.dir/build.make:76: modules/python3/CMakeFiles/opencv_python3.dir/__/src2/cv2.cpp.o] Error 1
  make[3]: Leaving directory '/test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build'
  make[2]: *** [CMakeFiles/Makefile2:2105: modules/python3/CMakeFiles/opencv_python3.dir/all] Error 2
  make[2]: Leaving directory '/test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build'
  make[1]: *** [Makefile:166: all] Error 2
  make[1]: Leaving directory '/test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build'
  Traceback (most recent call last):
    File "/tmp/pip-build-env-rmlqcdjw/overlay/lib/python3.8/site-packages/skbuild/setuptools_wrap.py", line 674, in setup
      cmkr.make(make_args, install_target=cmake_install_target, env=env)
    File "/tmp/pip-build-env-rmlqcdjw/overlay/lib/python3.8/site-packages/skbuild/cmaker.py", line 691, in make
      self.make_impl(clargs=clargs, config=config, source_dir=source_dir, install_target=install_target, env=env)
    File "/tmp/pip-build-env-rmlqcdjw/overlay/lib/python3.8/site-packages/skbuild/cmaker.py", line 736, in make_impl
      raise SKBuildError(msg)

  An error occurred while building with CMake.
    Command:
      /tmp/pip-build-env-rmlqcdjw/overlay/lib/python3.8/site-packages/cmake/data/bin/cmake --build . --target install --config Release --
    Install target:
      install
    Source directory:
      /test/opencv-python
    Working directory:
      /test/opencv-python/_skbuild/linux-x86_64-3.8/cmake-build
  Please check the install target is valid and see CMake's output for more information.

  error: subprocess-exited-with-error
  
  × Building wheel for opencv-python-headless (pyproject.toml) did not run successfully.
  │ exit code: 1
  ╰─> See above for output.
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
  full command: /test/venv/bin/python3 /test/venv/lib/python3.8/site-packages/pip/_vendor/pyproject_hooks/_in_process/_in_process.py build_wheel /tmp/tmpl8wghts_
  cwd: /test/opencv-python
  Building wheel for opencv-python-headless (pyproject.toml) ... error
  ERROR: Failed building wheel for opencv-python-headless
Failed to build opencv-python-headless
ERROR: Failed to build one or more wheels
```

</p></details>

So this time it did not even build OpenCV properly, failing to compile against Numpy.

--- a/find_version.py
+++ b/find_version.py
@@ -65,7 +65,7 @@ if __name__ == "__main__":
     else:
         # local version identifier, not to be published on PyPI
         version = git_hash
-        opencv_version += ".{}".format(version)
+        opencv_version += "+{}".format(version)
 
     with open("cv2/version.py", "w") as f:
         f.write('opencv_version = "{}"\n'.format(opencv_version))

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -4,7 +4,7 @@ requires = [
   "scikit-build>=0.13.2",
   "numpy==1.13.3; python_version=='3.6' and platform_machine != 'aarch64' and platform_machine != 'arm64'",
   "numpy==1.17.0; python_version=='3.7' and platform_machine != 'aarch64' and platform_machine != 'arm64'",
-  "numpy==1.17.3; python_version=='3.8' and platform_machine != 'aarch64' and platform_machine != 'arm64'",
+  "numpy==1.17.5; python_version=='3.8' and platform_machine != 'aarch64' and platform_machine != 'arm64'",
   "numpy==1.19.3; python_version<='3.9' and sys_platform == 'linux' and platform_machine == 'aarch64'",
   "numpy==1.21.0; python_version<='3.9' and sys_platform == 'darwin' and platform_machine == 'arm64'",
   "numpy==1.19.3; python_version=='3.9' and platform_machine != 'aarch64' and platform_machine != 'arm64'",

